src/server/dragonfly_test.cc

```diff
@@ -109,17 +109,13 @@ TEST_F(DflyEngineTest, Sds) {
 
 class DflyRenameCommandTest : public DflyEngineTest {
  protected:
-  DflyRenameCommandTest() : DflyEngineTest() {
+  DflyRenameCommandTest() {
     // rename flushall to myflushall, flushdb command will not be able to execute
     absl::SetFlag(
         &FLAGS_rename_command,
         std::vector<std::string>({"flushall=myflushall", "flushdb=", "ping=abcdefghijklmnop"}));
   }
-
-  void TearDown() {
-    absl::SetFlag(&FLAGS_rename_command, std::vector<std::string>({""}));
-    DflyEngineTest::TearDown();
```

Comment:
@abhijat because TearDown() is virtual, it gets called anyway at the end of the test scope right ? 

So this is fine :)

Reply:
Yes, `BaseFamilyTest::TearDown` ends up getting called as that's the only `TearDown` method that is defined in the inheritance chain.